controllers/volumereplicationgroup_controller.go

```diff
@@ -562,6 +553,10 @@ func (v *VRGInstance) updatePVCListForAll() error {
 
 	v.log.Info(fmt.Sprintf("Found %d PVCs using matching lables %v", len(pvcList.Items), labelSelector.MatchLabels))
 
+	if v.instance.Spec.Async == nil {
```

Comment:
This block cannot be here.  You'll end up with an empty volRepPVCs. Maybe change line 508 to look like this:
```
if v.instance.Spec.VolSync.Disabled || v.instance.Spec.Async == nil {
...
}
```

Reply:
Thanks @BenamarMk, I agree this is wrong.  And I agree with the proposed fix.  Another commit was pushed containing it.

It seems a synchronous replication test with VolSync.Disabled = false is missing.  Perhaps DRPC would never do that.  Otherwise, I would have expected the MetroDR tests to fail.